src/pages/HomePage/index.tsx

```diff
@@ -1,3 +1,26 @@
-const HomePage = () => <div />;
+import BalckButton from 'components/common/BalckButton';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const HomePage: React.FC = () => (
```

Comment:
리액트 컴포넌트라는 점을 명시해주지 않아도 자연스러울 것 같습니다!

```suggestion
const HomePage = () => (
```